app/frontend/packs/components/checkbox_search_filter.js

```diff
@@ -0,0 +1,202 @@
+const outerHeight = (el) => {
+  let height = el.offsetHeight
+  const style = window.getComputedStyle(el)
```

Comment:
FYI will break in IE8 and some other browsers based on only parsing one param
https://caniuse.com/?search=getComputedStyle